pkg/controller/nodelifecycle/scheduler/taint_manager_test.go

```diff
@@ -111,6 +108,63 @@ func createNoExecuteTaint(index int) v1.Taint {
 	}
 }
 
+// Waits for delete actions to appear and then returns them
+func getDeleteActions(fakeClientset *fake.Clientset) []clienttesting.Action {
+	var actions []clienttesting.Action
+main:
+	for ; ; actions = fakeClientset.Actions() {
+		for _, action := range actions {
+			if action.GetVerb() == "delete" {
+				break main
+			}
+		}
+	}
+	return actions
+}
+
+// Waits for delete actions to appear and checks if pod names
+// are equal to those stated in actions. When there appeared
+// all the actions with stated pod names then those actions returned
+func getDeleteActionsWithCertainPods(fakeClientset *fake.Clientset, podNames []string) []clienttesting.Action {
+	var actions []clienttesting.Action
+
+	var podActionsReceived []string
+	isEqual := func(a []string, b []string) bool {
```

Comment:
Use string sets instead of slices for the ease of comparison. You may find some examples in gc_controller_test.go

Reply:
Fixed